index.js

```diff
@@ -24,11 +24,17 @@ var ENV_IS_BROWSER = process.browser || false;
 /**
 * @param apiKey Giphy API key. Defaults to the public beta API key
 */
-var GiphyAPI = function(apiKey, options) {
-    options = options || {};
-    
-    this.apiKey = apiKey || PUBLIC_BETA_API_KEY;
-    this.timeout = options && options.timeout || 30000;
+var GiphyAPI = function(options) {
+    if (typeof options === 'string' || typeof options === 'undefined') {
+        this.apiKey = options || PUBLIC_BETA_API_KEY;
+        options = {};
+    } else if (typeof options === 'object') {
```

Comment:
This'd match `null` then throw on the next line, but then again, if someone does that, it's their own fault.